apps/condo/domains/billing/access/BillingReceipt.js

```diff
@@ -2,10 +2,34 @@
  * Generated by `createschema billing.BillingReceipt 'context:Relationship:BillingIntegrationOrganizationContext:CASCADE; importId?:Text; property:Relationship:BillingProperty:CASCADE; account:Relationship:BillingAccount:CASCADE; period:CalendarDay; raw:Json; toPay:Text; services:Json; meta:Json'`
  */
 
-const { canManageBillingEntityWithContext, canReadBillingEntity } = require('@condo/domains/billing/utils/accessSchema')
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+const { canManageBillingEntityWithContext } = require('@condo/domains/billing/utils/accessSchema')
 
-async function canReadBillingReceipts ({ authentication: { item: user } }) {
-    return await canReadBillingEntity(user)
+async function canReadBillingReceipts ({ context, authentication: { item: user } }) {
+
+    async function _getBillingAccountByUser (user) {
```

Comment:
move to utils

Reply:
it it's only for this file you can just move it at the top to avoid utilities overflow %)